.circleci/src/workflows/site-factory.yml

```diff
@@ -0,0 +1,18 @@
+jobs:
```

Comment:
@jmaupetit I'm moving this file inside a folder to allow git ignoring the workflow files related to sites.